docs/quant/2_browse_files.md

```diff
@@ -0,0 +1,73 @@
+
+### Browse Metadata
+
+You can browse the metadata for the reanalysis by clicking on **Browse Metadata** from the reanalysis webpage. This button is activated only when any tabular-type file(s) is available in the `Metadata` category.
+
+![](img/access_quant_reanalyses/browse_metadata_step1.png)
+
+You will move to the new webpage to show the list of files under the `Metadata` category.
+![](img/access_quant_reanalyses/browse_metadata_step2.png)
+
+
+Select one file name and it will show how the file looks. This is the example of an annotation file for MSstats, which requires *Run* or *Raw.file*, *Condition*, *BioReplicate*. It can have many additional columns.
```

Comment:
Some annotation files have both *Run* **and** *Raw.file*, I think it would be important to bring that up, and to indicate maybe that some in MassIVE.quant *Run* refers to the submitted filename and *Raw.file* refers to the filename uploaded?